packages/playground/remote/src/lib/offline-mode-cache.ts

```diff
@@ -69,11 +74,13 @@ export class OfflineModeCache {
 		 * The development environment uses Vite which doesn't work offline because it dynamically generates assets.
 		 * Check the README for offline development instructions.
 		 */
-		if (
-			url.href.startsWith('http://127.0.0.1:5400/') ||
-			url.href.startsWith('http://localhost:5400/') ||
-			url.pathname.startsWith('/website-server/')
-		) {
+		// @ts-ignore-next-line
+		if (import.meta.env.MODE === 'development') {
```

Comment:
Nice, I like that much better than the URL filter.

Reply:
I wanted to do that initially, but the PHP-wasm package where the code was didn't allow it.